### PR TITLE
fixing most E222 warnings in py files

### PR DESCRIPTION
**Pull request by fchapoton**
### :books: Description

fixing most E222 warnings issued by pycodestyle in `.py` files

E222 multiple spaces after operator

purely done with autopep8

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/graphs/graph_database.py [hunk 1]
```diff
@@ -180,18 +180,18 @@ def subgraphs_to_query(subgraphs, db):
 
 # tables     columns                    input data type     sqlite data type
 # -----------------------------------------------------------------------------
-aut_grp =  ['aut_grp_size',             # Integer           INTEGER
+aut_grp = ['aut_grp_size',             # Integer           INTEGER
             'num_orbits',               # Integer           INTEGER
             'num_fixed_points',         # Integer           INTEGER
             'vertex_transitive',        # bool              BOOLEAN
             'edge_transitive']          # bool              BOOLEAN
```
Comment (kwankyu) [inline src/sage/graphs/graph_database.py]: Well.. Then the following lines should be adjusted to be aligned right.

**Comment by kwankyu** [comment]
Otherwise, lgtm.

File: src/sage/graphs/graph_database.py (hunk 1 above)
Comment (fchapoton) [reply src/sage/graphs/graph_database.py]: indeed, done

**Review by kwankyu — APPROVED** [review]
Thanks.

**Pull request merged**